packages/shared/src/components/auth/AuthOptions.tsx

```diff
@@ -344,12 +351,12 @@ function AuthOptions({
       className={classNames(
         'z-1 flex w-full max-w-[26.25rem] flex-col overflow-y-auto rounded-16',
         !simplified && 'bg-theme-bg-tertiary',
-        className,
+        className?.container,
       )}
     >
       <TabContainer<AuthDisplay>
         onActiveChange={(active) => onSetActiveDisplay(active)}
-        controlledActive={activeDisplay}
+        controlledActive={forceDefaultDisplay ? defaultDisplay : activeDisplay}
```

Comment:
Fixed the `forceDefaultDisplay` to always show it, the primary reason we are forcing it, rather than having it in the initial state.